src/hooks/usePools.ts

```diff
@@ -105,8 +105,18 @@ export function usePools(
       if (!tokens) return [PoolState.INVALID, null]
       const [token0, token1, fee] = tokens
 
-      const { result: slot0, loading: slot0Loading, valid: slot0Valid } = slot0s[index]
-      const { result: liquidity, loading: liquidityLoading, valid: liquidityValid } = liquidities[index]
+      const {
```

Comment:
nit: Instead of a nullish operator, can you explicitly check for these? eg

```
if (!slot0s[index]) return [PoolState.INVALID, null]
```

This may be easier to read, but will also bring it in line with how tokens are checked directly above it.